pkgs/development/python-modules/qtile-extras/default.nix

```diff
@@ -21,28 +24,39 @@
 }:
 buildPythonPackage rec {
   pname = "qtile-extras";
-  version = "0.33.0";
+  version = "0.34.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "elParaguayo";
     repo = "qtile-extras";
     tag = "v${version}";
-    hash = "sha256-3aN2MrD1U5iBneVbYtNeWpK+JAQunGpemDpJnDuQdVI=";
+    hash = "sha256-CtmTZmUQlqkDPd++n3fPbRB4z1NA4ZxnmIR84IjsURw=";
   };
 
+  patches = [
+    # Remove unpack of widget.eval call in tests
+    # https://github.com/elParaguayo/qtile-extras/pull/460
+    (fetchpatch {
+      url = "https://github.com/elParaguayo/qtile-extras/commit/359964520a9dcd2c7e12680bfc53e359d74c489b.patch?full_index=1";
```

Comment:
Just FYI, the `?full_index=1` suffix makes no difference to the actual patch that is being applied, thanks to `fetchpatch` sanitizing the patch to a simple diff (that's way also the `.diff` prefix works the same).